src/test/java/org/isf/therapy/Tests.java

```diff
@@ -366,15 +366,14 @@ public void testTherapyRowToString() throws Exception {
 	}
 
 	@Test
-	public void testTherapyRowEquals() throws Exception {
+	void testTherapyRowEquals() throws Exception {
 		int id = setupTestTherapyRow(true);
 		TherapyRow foundTherapyRow = therapyIoOperationRepository.findById(id).orElse(null);
-		assertThat(foundTherapyRow).isNotNull();
-
 		assertThat(foundTherapyRow)
-				.isEqualTo(foundTherapyRow)
-				.isNotNull()
-				.isNotEqualTo("someString");
+			.isNotNull()
+			.isEqualTo(foundTherapyRow)
+			.isNotNull()
```

Comment:
Also here some redundancy

Reply:
Removed the duplicate `.isNotNull()`.   The `isEqualTo()` test is done a few lines lower and thus this one is removed.